aspen/resources/__init__.py

```diff
@@ -97,6 +97,52 @@ def get_declaration(line):
     return fulltext.decode(encoding or b'ascii')
 
 
+
+class SimplateWrapper(Simplate):
```

Comment:
I'm not sure this is the right class design for a fully-extracted `simplates.py`, but as a way disentangle simplates inside of Aspen I'm fine with it.